packages/beacon-node/src/chain/seenCache/seenGossipBlockInput.ts

```diff
@@ -164,7 +157,13 @@ export class SeenGossipBlockInput {
     if (!this.blockInputCache.has(blockHex)) {
       this.blockInputCache.set(blockHex, blockCache);
       callInNextEventLoop(() => {
-        this.reconstructColumns(config, blockHex);
+        getDataColumnsFromExecution(config, this.custodyConfig, this.executionEngine, this.emitter, blockCache)
+          .then((_success) => {
+            // TODO: (@matthewkeil) add metrics collection point here
+          })
+          .catch(() => {
+            // need to handle these errors so they don't percolate as uncaught and crash the node
```

Comment:
We should log this error here and not silently fail

Reply:
Pushed an error log for this, had to pass in a logger instance to SeenGossipBlockInput